casepro/statistics/signals.py

```diff
@@ -52,3 +53,24 @@ def record_incoming_labelling(sender, instance, action, reverse, model, pk_set,
     elif action == 'pre_clear':
         for label in instance.labels.all():
             DailyCount.record_removal(day, DailyCount.TYPE_INCOMING, label)
+
+
+@receiver(post_save, sender=CaseAction)
+def record_new_case_action(sender, instance, created, **kwargs):
+    """
+    This is where we keep track of DailyCounts for users within organisations
+    """
+    org = instance.case.org
+    user = instance.created_by
```

Comment:
In our latest PRs, we've added a `user_assignee` field to both the `Case` and `CaseAction` fields, which we use to show on the case view `user1 reassigned this case to partner2 (user2)`. I'm wondering what would be more useful, tracking this users who performed the action, or tracking who the case was assigned to.